lib/location/Kconfig

```diff
@@ -13,12 +13,16 @@ config LOCATION_METHOD_GNSS
 	bool "Allow GNSS to be used for obtaining the location"
 	depends on NRF_MODEM_LIB
 	depends on LTE_LINK_CONTROL
+	imply LTE_LC_NEIGHBOR_CELL_MEAS
+	imply LTE_LC_PSM
+	imply LTE_LC_MODEM_SLEEP
```

Comment:
@tejlmand
Do we have other options here than using `depend on` instead of `imply` (or `select`)?
I wouldn't like Location library users to need to define and know about the lte_lc modules that are required.